.github/workflows/UnitTestArm.yml

```diff
@@ -0,0 +1,36 @@
+name: Unit test for Arm
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version: ['1.0', '1', 'nightly']
```

Comment:
maybe just `1` to save CI build time? I'm not sure.

Reply:
Since those tests are run in parallel, I do not think reducing the targets will have much of an effect on speeding up the total process.